sample/src/main/kotlin/com/adevinta/spark/sample/BrowserApp.kt

```diff
@@ -132,9 +133,11 @@ internal fun ShowkaseBrowserApp(
             val navController = rememberNavController()
             val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior()
             val coroutineScope = rememberCoroutineScope()
-            val sheetState = rememberModalBottomSheetState(ModalBottomSheetValue.Hidden)
-            ModalBottomSheetLayout(
-                sheetState = sheetState,
+            val sheetState = rememberStandardBottomSheetState(Hidden)
+            val scaffoldState = rememberBottomSheetScaffoldState(bottomSheetState = sheetState)
+
+            BottomSheetScaffold(
```

Comment:
Not important as this part of the app will change but it should've been still `ModalBottomSheetLayout` instead of  `BottomSheetScaffold` here

Reply:
Yup, I changed it because I deprecated `ModalBottomSheetLayout` so a warning appears.
Do you want me to remove de deprecation ?

Reply:
No don't worry the new catalog app is almost done